tensorflow_probability/examples/jupyter_notebooks/Fitting_DPMM_with_Symmetric_Dirichlet_Prior_Using_pSGLD.ipynb

```diff
@@ -0,0 +1,975 @@
+{
+  "nbformat": 4,
+  "nbformat_minor": 0,
+  "metadata": {
+    "colab": {
+      "name": "Fitting DPMM with Symmetric Dirichlet Prior Using pSGLD",
+      "version": "0.3.2",
+      "provenance": [],
+      "collapsed_sections": [],
+      "toc_visible": true
+    },
+    "kernelspec": {
+      "name": "python3",
+      "display_name": "Python 3"
+    },
+    "accelerator": "TPU"
+  },
+  "cells": [
+    {
+      "metadata": {
+        "id": "NkfHUS813zJk",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "##### Copyright 2018 The TensorFlow Authors.\n",
+        "\n",
+        "Licensed under the Apache License, Version 2.0 (the \"License\");\n"
+      ]
+    },
+    {
+      "metadata": {
+        "id": "JC-lgcjL3892",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "cell_type": "code",
+      "source": [
+        "#@title Licensed under the Apache License, Version 2.0 (the \"License\"); { display-mode: \"form\" }\n",
+        "# you may not use this file except in compliance with the License.\n",
+        "# You may obtain a copy of the License at\n",
+        "#\n",
+        "# https://www.apache.org/licenses/LICENSE-2.0\n",
+        "#\n",
+        "# Unless required by applicable law or agreed to in writing, software\n",
+        "# distributed under the License is distributed on an \"AS IS\" BASIS,\n",
+        "# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.\n",
+        "# See the License for the specific language governing permissions and\n",
+        "# limitations under the License."
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "metadata": {
+        "id": "Um6EsGjz4RON",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "# Fitting Dirichlet Process Mixture Model with Symmetric Dirichlet Prior Using Preconditioned Stochastic Gradient Langevin Dynamics"
+      ]
+    },
+    {
+      "metadata": {
+        "id": "vsGdTAPr2FtM",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "In this notebook, we will demonstrate how to cluster a large number of samples and infer the number of clusters simultaneously by fitting Dirichlet Process Mixture of Gaussian distribution with Symmetric Dirichlet Prior using Preconditioned Stochastic Gradient Langevin Dynamics (pSGLD). "
```

Comment:
I might break this up in a few lines, by saying something like:

"by fitting a Dirichlet Process Gaussian Mixture Model. We use Preconditioned SGLD for inference."

The statement about what prior is being used can be stated below in the code / notes (and in fact I might remove the prior from the title itself).

Reply:
I broke the part into two lines as suggested and I dropped the "symmetric dirichlet prior" from title and file names.